.claude/skills/debug-e2e-tests/SKILL.md

```
@@ -0,0 +1,150 @@
+
+| Category                    | Description                                          | Where to fix     |
+| --------------------------- | ---------------------------------------------------- | ---------------- |
+| **A. Test code issue**      | Bad selector, missing wait, wrong assertion          | `e2e-tests/`     |
+| **B. UI bug**               | Component rendering error, broken user flow          | `src/`           |
+| **C. Infrastructure flake** | WebSocket drop, cluster timeout, resource exhaustion | Test resilience  |
+| **D. Pipeline/backend**     | Build failure, API error, auth problem               | Backend/config   |
+| **E. Cascade failure**      | Caused by an earlier test failing                    | Fix root failure |
+
+#### Report format
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)